job_definitions/index_services.yml

```diff
@@ -7,12 +7,19 @@
     parameters:
       - string:
           name: INDEX
-          default: {{ search_config[environment].default_index }}
-          description: "Index name to use (eg 'g-cloud-9-2017-05-22')"
+          default: {{ search_config['services'][environment].default_index }}
+          description: "Index name or alias to use (eg 'g-cloud-9-2017-05-22')"
       - string:
           name: FRAMEWORKS
-          default: {{ search_config[environment].frameworks }}
+          default: {{ search_config['services'][environment].frameworks }}
           description: "Comma-separated list of framework slugs that should be indexed (eg 'g-cloud-7,g-cloud-8'). If no frameworks are specified then all currently published services will be indexed."
+      - string:
+          name: CREATE_WITH_MAPPING
+          description: >
+            Specify a mapping to create the index if it doesn't exist. If it does
```

Comment:
I find this description a little hard to understand (it's probably because I know nothing about mappings). I don't quite get what you mean by 'under manual control'.